src/Search.js

```diff
@@ -1,30 +1,79 @@
 import React from "react";
 import "./App.css";
+import "./Search.css";
+import iFixitApi from './iFixitApi';
+import Autosuggest from 'react-autosuggest';
+import { FaSearch } from 'react-icons/fa';
+
+const getSuggestionValue = suggestion => suggestion.display_title;
+
+const renderSuggestion = suggestion => (
+	<span>{suggestion.display_title}</span>
+);
+
+const renderInputComponent = inputProps => (
+	  <div className="inputContainer">
+	    <FaSearch className="icon" />
+	    <input {...inputProps} />
+	  </div>
+);
 
 class SearchBar extends React.Component{
 	constructor(props){
 		super(props);
-		this.state = {searchString : ''};
+		this.state = {value : '', suggestions:[]};
 
-		this.handleChange = this.handleChange.bind(this);
+		this.onChange = this.onChange.bind(this);
 		this.handleSubmit = this.handleSubmit.bind(this);
+		this.onSuggestionsFetchRequested = this.onSuggestionsFetchRequested.bind(this);
+		this.onSuggestionsClearRequested = this.onSuggestionsClearRequested.bind(this);
 	}
 
-	handleChange(event){
-		this.setState({searchString : event.target.value});
-	}
+	onChange = (event, { newValue }) => {
+		this.setState({ value: newValue	});
+	};
+
+	async onSuggestionsFetchRequested({ value }){
+		let newSuggestions = await iFixitApi.get(`suggest/${value}?doctypes=category`).then(response => {
+			return Array.from(response.data.results, device => {return { 'display_title': device.display_title, 'id': device.wikiid}}); 
```

Comment:
Arrow functions implicitly return values so you can get rid of the return call to clean this up a little bit. You just have to wrap the braces in parenthesis since you are returning an object literal.
```suggestion
			return Array.from(response.data.results, device => ({ 'display_title': device.display_title, 'id': device.wikiid})); 
```

Reply:
Also no need for using strings as the object fields since they don't contain any special characters
```suggestion
			return Array.from(response.data.results, device => ({ display_title: device.display_title, id: device.wikiid})); 
```

Reply:
I'm assuming `response.data.results` is an array. If so you can just use the `.map` function which is more common than using `Array.from`. So all suggestions together:
```suggestion
			return response.data.results.map(device => ({ display_title: device.display_title, id: device.wikiid}); 
```